htdocs/product/list.php

```diff
@@ -458,6 +458,10 @@
 	if($type == Product::TYPE_SERVICE) $rightskey='service';
 	if($user->rights->{$rightskey}->creer)
 	{
+		if ($type === "") {
+			$newcardbutton.= dolGetButtonTitle($langs->trans('NewProduct'), '', 'fa fa-plus-circle', DOL_URL_ROOT.'/product/card.php?action=create&amp;type=0');
```

Comment:
What is goal of this line. $newcardbutton is erased just after ?

Reply:
This line adds a **Add new Product** action button if the type query string parameter is not defined and then define the $type variable to `Product::TYPE_SERVICE` to display also the **Add new Service**.

`$newcardbutton` is not erased just after because of the use of `.=` (concatenation operator).

Reply:
So if type is "", we will get 2 buttons to create a product ?